### PR TITLE
Move OldTests to xUnit and into Tests Project

### PR DESCRIPTION
**Pull request by JVanderford**

### CODE REVIEW

**Comment by JVanderford** [comment]
@BWortman  The changes you requested are not committed up.

**Comment by JVanderford** [comment]
Thanks, @JeremiahDotNet   I'll update the solution and rework the tests to follow the single assertion pattern.

**Comment by JeremiahDotNet** [comment]
I was looking closer at the tests and it looks like they are creating Mocks of the Subject Under Test instead of testing the actual class.

`TokenManager` should be a new instance and it's dependencies should be mocked. I suspect there will be more work needed to verify `TokenManager` and the `AuthenticateUser` classes are working correctly.  I think we should open two more issues to debug/verify both classes and to update the content of their unit tests.

Everything else looks good.

**Comment by JVanderford** [comment]
@JeremiahDotNet  I see what you're talking about, I should be able to take care of that rather quickly in the morning.

**Comment by JVanderford** [comment]
I've updated the logincontroller and authenticateUser tests,  The tests on the TokenManager look a little more complicated to update, so I'll get to them in the morning.

File: AutoRenter.Api.Tests/AuthenticateUserTests.cs [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+using AutoRenter.Api.Models;
+
+namespace AutoRenter.Api.Tests
+{
+    public class AuthenticateUserTests
+    {
+        [Fact]
+        public void Execute_ShouldReturnNullDataInResultModel()
+        {
+            //assemble
```
Comment (JeremiahDotNet) [inline AutoRenter.Api.Tests/AuthenticateUserTests.cs]: The other tests use the word arrange instead of assemble. Not a big deal but nice for uniformity.

**Comment by JeremiahDotNet** [comment]
@BWortman Everything looks good to me. You can do your review and/or merge the changes when you're ready.

**Comment by JeremiahDotNet** [comment]
Brian's requested changes have been met. Dismissing the review to allow the PR to be merged.

**Pull request merged**